doc/api/vm.md

```diff
@@ -920,6 +920,33 @@ to disallow any changes to it.
 Corresponds to the `[[RequestedModules]]` field of [Cyclic Module Record][]s in
 the ECMAScript specification.
 
+### `sourceTextModule.getHasAsyncGraph()`
```

Comment:
Does it need to be a method? It seems fine to just make it a getter property, and it would be more consistent with other getter properties.

Reply:
This requires the module to be instantiated, and throws before instantiation.

I think throwing in a getter is not a good pattern. Making it a method indicates that this could incur traversal on the dependency graph, and may throw if this is not instantiated.